sdk/monitor/opentelemetry-exporter-azure-monitor/src/export/trace.ts

```diff
@@ -31,21 +31,16 @@ export class AzureMonitorTraceExporter implements SpanExporter {
 
   constructor(options: Partial<AzureExporterConfig> = {}) {
     const connectionString = options.connectionString || process.env[ENV_CONNECTION_STRING];
-
     this._logger = new ConsoleLogger(LogLevel.ERROR);
-    this._options = {
-      ...DEFAULT_EXPORTER_CONFIG,
-      ...options
-    };
+    this._options = DEFAULT_EXPORTER_CONFIG;
+    this._options.serviceApi = options.serviceApiVersion ?? this._options.serviceApi;
 
     if (connectionString) {
       const parsedConnectionString = ConnectionStringParser.parse(connectionString, this._logger);
-      this._options = {
-        ...DEFAULT_EXPORTER_CONFIG,
-        // Overwrite options with connection string results, if any
-        instrumentationKey: parsedConnectionString.instrumentationkey ?? "",
-        endpointUrl: parsedConnectionString.ingestionendpoint ?? ""
-      };
+      this._options.instrumentationKey =
```

Comment:
Looks like you used to "clone" DEFAULT_EXPORTER_CONFIG but now you're directly assigning it. Won't that cause these assignments below to overwrite the global DEFAULT_EXPORTER_CONFIG? Or is that intentional?